ui/webui/test/check-storage

```diff
@@ -109,8 +109,8 @@ class TestStorage(anacondalib.VirtInstallMachineCase, StorageHelpers):
         s.modify_storage()
         # Run the tool
         b.click("#modify-storage-modal-modify-btn")
-        b.wait_visible(f"#modify-storage-modal-modify-btn:not([aria-disabled={True}]")
-        b.wait_visible(f"#storage-modified-modal-rescan-btn")
+        b.wait_visible("#modify-storage-modal-modify-btn:not([aria-disabled=false]")
```

Comment:
This is pretty useless. Before clicking the button should be always enabled, then get disabled and then re-enabled. So now you 're checking either the pre-disabled state, or the post-disabled state. this is not deterministic, I think we can drop it.

Reply:
I think it is checking the disabled state (I read it disabled is not false)